### DISCUSSION TITLE
Support for additional deck.gl layers like terrain layer or h3 clusters?

### DISCUSSION THREAD

**Discussion by cboettig** [developmentseed/lonboard#493] [Q&A]
Is it already possible to interleave other deck.gl layers with lonboard's maps?  E.g. can we add a [deckgl terrain layer](https://deckgl.readthedocs.io/en/latest/gallery/terrain_layer.html) or a [deck's h3 clusters](https://deckgl.readthedocs.io/en/latest/gallery/h3_cluster_layer.html) or do these need to be implemented first as Lonboard layers?

**Comment by kylebarron** [comment]
They need to be implemented first as Lonboard layers to provide validation for input parameters and connect to the javascript layer description. PRs are welcome, and I can point you to how to implement additional layers. E.g. https://github.com/developmentseed/lonboard/issues/302#issuecomment-1874423165
